src/process/data-provider.js

```diff
@@ -5,23 +5,41 @@ import {DateTime} from "luxon";
 
 const from = `${parseInt(Config.PROCESS_WINDOW) - 1} HOURS`;
 const to = `${Config.PROCESS_WINDOW} HOURS`;
+
 const enrichedDataQuery = `
-    select * from enriched_data
+    select *
+    from enriched_data
```

Comment:
To może być razem, bo jest dosyć krótkie

Reply:
Co innego gdybyś wymieniał co ma być wybrane w select. Tutaj jest * więc nie ma co tak wydłużać kod